web/src/components/network/IpPrefixInput.tsx

```diff
@@ -20,17 +20,26 @@
  */
 
 import React, { useState } from "react";
-import { isValidIpPrefix } from "~/client/network/utils";
-import { TextInput, ValidatedOptions } from "@patternfly/react-core";
+import { isValidIpPrefix } from "~/utils/network";
+import { TextInput, TextInputProps, ValidatedOptions } from "@patternfly/react-core";
 import { _ } from "~/i18n";
 
-const IpPrefixInput = ({ placeholder, onError = () => null, ...props }) => {
-  const [validated, setValidated] = useState("default");
+const IpPrefixInput = ({
+  label = _("Ip prefix or netmask"),
+  onError = (value) => null,
+  ...props
+}: TextInputProps & { defaultValue?: string | number }) => {
+  const [validated, setValidated] = useState(
+    !props.defaultValue || props.defaultValue === "" || isValidIpPrefix(props.defaultValue)
```

Comment:
same applies here...maybe having it in own method can help like `emptyOrValidIPrefix(props.defaultValue)` or maybe even with returned string something like `IpPrefixState(props.defaultValue)`

Reply:
Let's try to improve it, thanks for commenting it :)